Source/RTS/Plugins/RTSPlugin/Source/RTSPlugin/Classes/RTSMinimapWidget.h

```diff
@@ -3,6 +3,7 @@
 #include "RTSPluginPCH.h"
 
 #include "Blueprint/UserWidget.h"
+#include "Launch/Resources/Version.h" // Included only for IntelliSense.
```

Comment:
If we don't include this here IntelliSense for some reason picks the wrong blocks to show as valid.